docs/CODE_STYLE.md

```
@@ -1,60 +1,85 @@
   The exception is for enum entries.
 
-## Recommended:
+## Recommended
 
-### File names:
+### File names
 Follow the general GameMaker convention of using type prefixes in file names.
 
-### Names:
-All variable names, function names, etc., should use `snake_case` unless otherwise specified.
+### Variable names
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)